src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write your code here
+const body = document.querySelector('body');
+
+const promise1 = new Promise((resolve) => {
+  const logo = document.querySelector('.logo');
+
+  logo.addEventListener('click', () => {
+    const resolveBlock = `
+    <div class="message">
+      Promise was resolved!
+    </div>`;
+
+    resolve(body.insertAdjacentHTML('beforeend', resolveBlock));
```

Comment:
Try not to invoke function in a function here. Good idea would be to for example pass a message in resolve() function like:
```js
resolve('Promise was resolved!');
```

And then you could create a showMessage function and invoke it in the success handler:
```js
promise1.then(res => {
  showMessage(res);
});
```

`showMessage` function could be also used to display error message based on the parameters

Reply:
Okay, For sure ;). Thank you for your comments!